### ISSUE TITLE
Update audioplayers to use path_provider v0.5.0

### ISSUE THREAD

**Issue by IrosTheBeggar** [luanpotter/audioplayers#108]
I tried updating my project to use path_provider v0.5.0 and go this error:

```
Running "flutter packages get" in mstream_flutter...            
Because audioplayers 0.8.0 depends on path_provider ^0.4.1 and no versions of audioplayers match >0.8.0 <0.9.0, audioplayers ^0.8.0 requires path_provider ^0.4.1.
So, because mstream_flutter depends on both audioplayers ^0.8.0 and path_provider ^0.5.0, version solving failed.

pub get failed (1)
exit code 1
```

Looks like dart doesn't handle dependencies as flexibly as node

**Comment by nemoryoliver** [comment]
This is a must too... Please.. thank you dev

**Comment by dukaric1991** [comment]
I am also dealing with the same problem...

**Comment by apiraino** [comment]
:+1: 

If no one steps in I'll try to contribute a PR this weekend (Flutter newbie here, but hey what could possibily go wrong)

**Comment by antoninobajeli** [comment]
I'm waiting too for this upgrade.
Cheers :)

**Comment by luanpotter** [comment]
This has been released on 0.8.2 ;)
